layers/cmd/valuesetcache.go

```diff
@@ -0,0 +1,57 @@
+package cmd
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"sort"
+
+	"github.com/cloudprivacylabs/lsa/pkg/ls"
+	lru "github.com/hashicorp/golang-lru/v2"
+)
+
+const cache_size = 65000
+
```

Comment:
Make a ValuesetCache interface with get/set
Have two impls, one that doesn't do anything, and the other is this LRU cache,